okhttp/src/main/java/okhttp3/Headers.java

```diff
@@ -48,6 +49,15 @@
  * <p>Instances of this class are immutable. Use {@link Builder} to create instances.
  */
 public final class Headers {
+  // TODO(oldergod) make them String ?
+  // Special header names defined in HTTP/2 spec.
+  public static final ByteString PSEUDO_PREFIX = ByteString.encodeUtf8(":");
+  public static final ByteString RESPONSE_STATUS = ByteString.encodeUtf8(":status");
+  public static final ByteString TARGET_METHOD = ByteString.encodeUtf8(":method");
+  public static final ByteString TARGET_PATH = ByteString.encodeUtf8(":path");
+  public static final ByteString TARGET_SCHEME = ByteString.encodeUtf8(":scheme");
+  public static final ByteString TARGET_AUTHORITY = ByteString.encodeUtf8(":authority");
```

Comment:
Don't think these make sense here. This API works on strings, not byte strings. Plus the values of these are never exposed publicly. What about on the `Http2` class?

Reply:
Looking at this now. There is some places there are `String` comparison, sometimes `ByteString` comparison; trying to find the right line to draw.

Reply:
Gave it a shot under Http2; still had to be public to be used inside `MockWebServer`.